src/commands/jeopardy.js

```diff
@@ -145,6 +145,16 @@ function isAnswerCorrect(message, answer) {
     text,
     answer.toLowerCase()
   );
+
+  // check if the user's submission matches the question's alternative answer (if any)
```

Comment:
this could be written to check the answer in parentheses, and check the answer excluding the parentheses